### ISSUE TITLE
>home< missing url

### ISSUE THREAD

**Issue by ghost** [cod1ng-earth/filDID#9]
>home< in left corner, missing a url.
See scrnshot
![2020-08-18 (2)](https://user-images.githubusercontent.com/61500781/90561741-78269480-e1a1-11ea-8523-58cb6bb864d8.png)

![2020-08-18](https://user-images.githubusercontent.com/61500781/90561522-1e25cf00-e1a1-11ea-9cda-b85f205536b8.png)
![2020-08-18 (1)](https://user-images.githubusercontent.com/61500781/90561936-bde35d00-e1a1-11ea-8b40-1bb280020d8a.png)

<a href="https://">Home</a>

**Comment by sylar217** [comment]
Yeah, I noticed this. Thanks will fix this as soon as I am able to jump on an update to ceramic doc implementation
